### PR TITLE
Add API Docs for Tax and Social Links

### LINKED ISSUE #3732: Implement API Blueprint for documenting the nextgen API
Follow up to discussion on #3641

We will be using Api-Blueprint (https://apiblueprint.org/). This issue is a high priority one. Needs to be done by the end of this week.

### LINKED ISSUE #3774: Dasherize all fields in the Api-blueprint Documentation
Dasherize all fields in the api blueprint in accordance with #3753

### PR DESCRIPTION
**Pull request by poush**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [x] The unit tests pass locally with my changes 

#### Short description of what this resolves:
Work Continued on API Documentation. Added Tax API docs in the Blueprint.
Preview can be seen here: http://docs.orgaserver.apiary.io/#

Partly Fixes #3732 
Fixes #3774

### CODE REVIEW

File: docs/api/api_blueprint.apib [hunk 1]
```diff
@@ -976,3 +976,260 @@ The Open Event API Server
 ### Delete Sponsor [DELETE]
 
 + Response 204
+
+## Tax Collection [/events/{event_id}/tax?page,limit]
++ Parameters
+    + event_id (integer) - ID of the spcific event in integer
```
Comment (enigmaeth) [inline docs/api/api_blueprint.apib]: `specific` :P

Comment (poush) [inline docs/api/api_blueprint.apib]: thanks :P

File: docs/api/api_blueprint.apib [hunk 2]
```diff
@@ -25,7 +25,7 @@ The Open Event API Server
 + Response 200 (application/json)
 
         {
-            "access_token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZGVudGl0eSI6MSwiaWF0IjoxNDk2OTU2ODMwLCJuYmYiOjE0OTY5NTY4MzAsImV4cCI6MTQ5NzA0MzIzMH0.bjl0"
+            "access-token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZGVudGl0eSI6MSwiaWF0IjoxNDk2OTU2ODMwLCJuYmYiOjE0OTY5NTY4MzAsImV4cCI6MTQ5NzA0MzIzMH0.bjl0"
```
Comment (poush) [inline docs/api/api_blueprint.apib]: I guess this will be dasherized

Comment (SaptakS) [inline docs/api/api_blueprint.apib]: Yes. This won't be dasherized as of now.

File: docs/api/api_blueprint.apib [hunk 3]
```diff
@@ -5,7 +5,7 @@ HOST: http://nextgen.eventyay.com
 
 The Open Event API Server
 
-## JWT Auth Key [/auth/session?filter_parameters]
+## JWT Auth Key [/auth/session?filter-parameters]
```
Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Auth endpoint has not GET params

Comment (SaptakS) [inline docs/api/api_blueprint.apib]: @niranjan94 this has been fixed in PR by @magdalenesuo [here](https://github.com/fossasia/open-event-orga-server/pull/3775). So can we merge this?

Comment (SaptakS) [inline docs/api/api_blueprint.apib]: @niranjan94 I am dismissing this review as discussed over gitter.

**Pull request merged**